.travis.yml

```diff
@@ -3,6 +3,7 @@ dist: focal
 language: ruby
 rvm:
 - ruby-head
```

Comment:
head = 3 at the moment, but head is allowed to fail. not sure if I should keep 3.0.0 here?